src/data/units.ts

```diff
@@ -3927,8 +3977,9 @@ export const unitData = [
 		id: 50,
 		name: 'Shadow Leech',
 		playable: false,
-		level: '1',
+		level: 1,
```

Comment:
Single quotes missing on this one.

Reply:
Are the single quotes intentional? The only other unit that has a `string` level is Dark Priest `level: '-'`. All other units have a `number` level

Reply:
Seemed lighter and easier to type I guess.

Reply:
I changed it to a `number` as it made the `Level` type defined in `units.ts` cleaner and I think easier to understand.

before : `type Level =  "-" | 1 | 3 | 2 | 4 | 5 | 6 | 7 | "1"`
after: `type Level = "-" | 1 | 3 | 2 | 4 | 5 | 6 | 7 `

This also simplifies type checking once these types are implemented in `game.ts`, as now if `Level` is a string the only possibility is `'--'` versus ` '--' | '1' `

However, it's your call, I'll just wait for clarification and then close this PR